data/data/aws/route53/variables.tf

```diff
@@ -54,3 +54,15 @@ variable "api_internal_lb_zone_id" {
   type        = string
 }
 
+variable "publish_strategy" {
+  type        = string
+  default     = "External"
+  description = <<EOF
+The publishing strategy for endpoints like load balancers
+
+Because of the issue https://github.com/hashicorp/terraform/issues/12570, the consumers cannot count 0/1
+based on if api_external_lb_dns_name for example, which will be null when there is no external lb for API.
+So publish_strategy serves an coordinated proxy for that decision.
+
```

Comment:
nit: empty line before the EOF

Reply:
fixed: https://github.com/openshift/installer/compare/09284a49abb5433054f290c35e0aa14a98eef974...2bd7e958c24af56fb51e653786ea3ad37f6d25f4